exec/java-exec/src/main/java/org/apache/drill/exec/physical/impl/aggregate/HashAggTemplate.java

```diff
@@ -278,7 +281,7 @@ public void setup(HashAggregate hashAggrConfig, HashTableConfig htConfig, Fragme
   }
 
   @Override
-  public AggOutcome doWork() {
+  public AggOutcome doWork() throws SchemaChangeException {
```

Comment:
You are right that Streaming aggregator can not handle schema change exception currently. 

In the current code since Drill's planner put sort operator before streaming aggregator, if the input has schema change, it would hit schema change exception in sort operator first, before come to streaming aggregator.

Anyway, I modified the code in StreamingAggregator, in case Drill planner generate a plan without sort for streaming aggregator.

I also simplified the change, since I realized we do not have to change in AggTemplate; the detailed error message could be produced in AggBatch.